app/routes.js

```diff
@@ -66,6 +66,7 @@ router.use((req, res, next) => {
     const isHardStop = (formdata, journey) => config.hardStopParams[journey].some(param => get(formdata, param) === commonContent.no);
     const executorsWrapper = new ExecutorsWrapper(formdata.executors);
     const hasMultipleApplicants = executorsWrapper.hasMultipleApplicants();
+    const skipPayment = get(formdata, 'payment.total') === 0 || get(formdata, 'payment.status') === 'Success';
```

Comment:
I realise the skipPayment makes reading the if/then/else statement a bit easier however, the if/then/else statement has this type of logic "inline".  We should either keep the code as the rest is or change the others to be like skipPayment - otherwise we have a mismatch of coding practices